README.md

```diff
@@ -1,92 +1,199 @@
+                           fptype = "float",
+                           method = "randomDense",
+                           distributed = True) # <-- change model to distributed
+
+centroids = init_alg.compute(data).centroids
+
+alg = d4p.kmeans(nClusters = 10, maxIterations = 50, fptype = "float",
+                 accuracyThreshold = 0, assignFlag = False,
+                 distributed = True)  # <-- change model to distributed
+
+result = alg.compute(data, centroids)
+```
+
+## Scikit-learn patching
+daal4py also have an API which matches API from scikit-learn.
+This framework allows you to speed up your existing projects by changing one line of code
+
+```python
+from daal4py.sklearn.svm import SVC
+from sklearn.datasets import load_digits
+
+digits = load_digits()
+X, y = digits.data, digits.target
+
+svm = SVC(kernel='rbf', gamma='scale', C = 0.5).fit(X, y)
+print(svm.score(X, y))
+```
+
+In addition daal4py provides an option to replace some scikit-learn methods by daal solvers which makes it possible to get a performance gain **without any code changes**. This approach is the basis of Intel distribution for python scikit-learn. You can patch stock scikit-learn by using the only following commandline flag
+```bash
+python -m daal4py my_application.py
+```
+Patches can also be enabled programmatically:
+```python
+from sklearn.svm import SVC
+from sklearn.datasets import load_digits
+
+svm_sklearn = SVC(kernel="rbf", gamma="scale", C=0.5)
+
+start = time()
+svm_sklearn = svm_sklearn.fit(X, y)
+end = time()
+print(start-end) # output: 0.141261...
+print(svm_sklearn.score(X, y)) # output: 0.9905397885364496
 
-## Examples
+from daal4py.sklearn import patch_sklearn
+patch_sklearn() # <-- apply patch
+from sklearn.svm import SVC
 
-Examples for different programming languages:
+svm_d4p = SVC(kernel="rbf", gamma="scale", C=0.5)
 
-- [C++](https://github.com/oneapi-src/oneDAL/tree/master/examples/cpp)
-- [C++ SYCL*](https://github.com/oneapi-src/oneDAL/tree/master/examples/cpp_sycl)
-- [Java](https://github.com/oneapi-src/oneDAL/tree/master/examples/java)
+start = time()
+svm_d4p = svm_d4p.fit(X, y)
+end = time()
+print(start-end) # output: 0.032536...
+print(svm_d4p.score(X, y)) # output: 0.9905397885364496
+```
+
+
+For more details browse our [daal4py documentation](https://intelpython.github.io/daal4py/).
+
+
+## DAAL Samples
+
+<img align="right" style="display:inline;" height=300 width=550 src="https://github.com/PivovarA/daal/raw/dev/apivovar-newReadme/.github/charts/intel%20DAAL%20Spark%20samples%20vs%20Apache%20Spark%20MLlib.png"></a>
+
+Samples is a non official part of a project, that contains examples of how DAAL can be used in different applications.
+
+**DAAL Spark Samples is a good example.**
+DAAL provides scala / java interfaces that match Apache Spark MlLib API and use DAAL solvers under the hood. This implementation allows you to get a 3-18X increase in performance compared to default Apache Spark MLlib.
```

Comment:
Some footnote is required for such a performance claim: hardware specs, datasets, software versions, etc.

Reply:
Yes, thanks for pointing that out. I have added descriptions to the charts with HW, SW and other details.